grant/projects/evolutionland.md

```diff
@@ -0,0 +1,118 @@
+
+### **Team**
+
+- Tiny Liu ( Project Manager of Evolution Land )
+- Shell Wan ( Front-end Engineer )
+- Perror Qin ( Backend Engineer )
+- Echo Hu ( Solidity Engineer)
+- Xinyue Wang ( UI/UX Designer )
+
+### **Legal Structure**
+
+Currently Evolution Land has no defined legal structure other then a Dapp created by Evolution Land team.
+
+## **Team Code Repos**
+
+- [https://github.com/evolutionlandorg](https://github.com/evolutionlandorg) (Evolution Land)
+
+## **Collaborative Grant Proposal and Current Development**
+
+### **Proposal**
+
+1. Build and deploy Evolution Land on Crab Network: a **Metaverse+Gamefi+cross-chain game**. 
+2. Start marketing events on Crab continent - Columbus.
+
+Evolution Land will provide the following to Darwinia and Crab community:
+
+- Latest version including PVE, equipment, and profession gameplay
+- Continously update on Columbus continent
+- Farm pool for users to stake $xRING
+- 70% of profit will return to users
+
+## Evolution Land **Beta Version:**
+
+The beta version of Evolution Land is what is applying for CRAB grant program, and the delivery scope has:
+
+- Basic mining gameplay including drills and apostles mining functions
+- Apostle Adventure - PVE gameplay
+- Furnance System - First NFT + FT Gamefi gameplay
+- Buy, sell, destroy and send functions
+- Apostle breeding system
+- Profession system
+
+## **Future Development Roadmap**
+
+The following milestones are not included in the collaboration grant assessment and is only here to communicate Evolution Land team's vision and direction.
+
+### Evolution Land **1.0**
+
+- Basic mining gameplay including drills and apostles mining functions
+- Apostle Adventure - PVE gameplay
+- Furnance System - First NFT + FT Gamefi gameplay
+- Buy, sell, and send functions
+- Apostle breeding system
+- Profession system
+
+### Evolution Land **2.0:**
+
+- Liquidity pools and farm pools
+- DAO
+
+### Evolution Land **3.0:**
+
+- PVP gameplay
+
+### Evolution Land **4.0:**
+
+- Building system
+
+## **Colabriation Funding**
+
+Total funding amount: 6,000 USDT. Payment to be made in two instalments to the value of $ 3,000 USD.
+
+The first installment is paid in RING to the value of $3,000 USD. The ratio of RING to determined by average rate of the prior week, set by the time of accepting the pull request. Price ratio to be generated by coinmarket.com or coingecko.com - if a automatited 7 day average is obtainable. The 7 day high + 7 day low / 2 is also acceptable - allowing for 8 points behind the decimal at each 7 day price point.
+
+The second payment is paid in USDT to the value of $3,000 USD on completion of the acceptance criteria judged by Darwinia.
+
+## **Additional Information**
+
+To trial our pre-beta product please go to: Evolution Land. You will need the following:
```

Comment:
Columbus is redirected to https://www.reddit.com/r/EvolutionLand/?

Reply:
Columbus land is not yet available for production version, https://www.evolution.land.l2me.com/land/3 is a beta version.